docs/audit-log-filter-variables.md

```diff
@@ -0,0 +1,745 @@
+* `audit_log_filter_rotate_on_size`
+* `audit_log_filter_prune_seconds`
+
+
+### `audit_log_filter_password_history_keep_days`
+
+| Option name | Description |
+| --- | --- |
+| Command-line | --audit-log-filter-password-history-keep-days |
+| Dynamic | Yes  |
+| Scope | Global  |
+| Data type | Integer  |
+| Default | 0  |
+
+Defines when passwords are may be removed and measured in days.
+
+Encrypted log files have passwords stored in the keyring. The plugin also stores a password history. A password does not expire, despite being past the value, in case the password is used for rotated audit logs. The operation of creating a password also archives the previous password.
+
+The default value is 0 (zero). This value disables the expiration of passwords. Passwords are retained forever. 
+
+If the plugin starts and encryption is enabled, the plugin checks for an audit log filter encryption password. If a password is not found, the plugin generates a random password.
+
+Call `audit_log_filter_encryption_set() to set a specific password.
+
+
+### `audit_log_filter_prune_seconds`
+
+| Option name | Description |
+| --- | --- |
+| Command-line | --audit-log-filter-prune-seconds |
+| Dynamic | Yes  |
+| Scope | Global  |
+| Data type | Integer  |
+| Default | 0  |
+| Minimum value | 0 |
+| Maximum value | 1844674073709551615 |
+| Unit | seconds |
+
+Defines when the audit log filter file is pruned. This pruning is based on the age of the file. The value is measured in seconds. 
+
+A value of 0 (zero) is the default and disables pruning. The maximum value is 18446744073709551615.
+
+A value greater than 0 enables pruning. An audit log filter file can be pruned after this value.
+
+To enable log pruning, you must set one of the following:
+
+* Enable log rotation by setting `audit_log_filter_rotate_on_size
+* Add a value greater than 0 (zero) for either `audit_log_filter_max_size` or `audit_log_filter_prune_seconds`
+
+
+### `audit_log_filter_read_buffer_size`
+
+| Option name | Description |
+| --- | --- |
+| Command-line | --audit-log-filter-read-buffer-size |
+| Dynamic | Yes  |
+| Scope | Global  |
+| Data type | Bytes  |
+| Default | 32768  |
+
+This option is only supported for JSON-format files.
+
+The size of the buffer for reading from the audit log filter file. The `audit_log_filter_read()` reads only from this buffer size.
+
+
+### `audit_log_filter_rotate_on_size`
+
+| Option name | Description |
+| --- | --- |
+| Command-line | --audit-log-filter-rotate-on-size |
+| Dynamic | Yes  |
+| Scope | Global  |
+| Data type | Integer |
+| Default | 1GB  |
+
+Performs an automatic log file rotation based on the size. The default value is 0 (zero).
+
+If the value is 1GB, the plugin does not automatically rotate the log file. If you set the value to less than 4096 resets the value to 0, and does not automatically rotate the log files. You can rotate the log files manually. The recommended way is to use [`audit_log_rotate()`](#audit_log_rotate).
```

Comment:
Shall the following be removed - "If the value is 1GB, the plugin does not automatically rotate the log file" ?Looks like it just duplicates the following sentence but with wrong logic.